stdlib/3/socket.pyi

```diff
@@ -250,6 +252,68 @@ TIPC_WITHDRAWN = 0
 TIPC_ZONE_SCOPE = 0
 
 
+# enum versions of above flags py 3.4+
+if sys.version_info >= (3, 4):
+    class AddressInfo(IntEnum):
+        AI_ADDRCONFIG = ...
```

Comment:
Confirmed locally that you don't need types for the values, at least with mypy. (So no action required from you.)